### DISCUSSION TITLE
help for badge

### DISCUSSION THREAD

**Discussion by oceaneRoche** [recodehive/Stackoverflow-Analysis#478] [Q&A] [answered]
https://github.com/oceaneRoche/pair-extraordinaire-badge

Please give me stars

**Comment by techy4shri** [comment]
are you perhaps aiming for pair-extraordinate badge?

**Reply by oceaneRoche** [reply, marked as answer]
no I'm aiming for the starstruck badge :)

**Comment by marv1n-le** [comment]
okay this is ez

**Reply by oceaneRoche** [reply]
thanks

**Reply by ghost** [reply]
Bro Please give me a star to achieve the badge
https://github.com/mte-tonmoy/mte-tonmoy
&  I will also give you star on your repository.

**Comment by TatyOko28** [comment]
I just starred your repository! ⭐ Keep up the great work, and I hope you reach your Galaxy badge soon! 🚀

**Reply by ghost** [reply]
Bro Please give me a star to achieve the badge
https://github.com/mte-tonmoy/mte-tonmoy
&  I will also give you star on your repository.

**Comment by sandipanrakshit34** [comment]
hey please give me a star on this repository and help me to achieve the badge:

https://github.com/sandipanrakshit34/sandipanrakshit34

**Reply by ghost** [reply]
Bro Please give me a star to achieve the badge
https://github.com/mte-tonmoy/mte-tonmoy
&  I will also give you star on your repository.

**Comment by pathipat11** [comment]
Hey please can someone help me get help me to achieve the badge.

**Reply by ghost** [reply]
Bro Please give me a star to achieve the badge
https://github.com/mte-tonmoy/mte-tonmoy
&  I will also give you star on your repository.

**Comment by indranil143** [comment]
please give me stars : )

https://github.com/indranil143/Real-vs.-AI-Generated-Face-Classification-using-Deep-Learning

**Reply by ghost** [reply]
Bro Please give me a star to achieve the badge
https://github.com/mte-tonmoy/mte-tonmoy
&  I will also give you star on your repository.

**Comment by ghost** [comment]
> https://github.com/oceaneRoche/pair-extraordinaire-badge
> 
> Please give me stars

Bro Please give me a star to achieve the badge
https://github.com/mte-tonmoy/mte-tonmoy
&  I will also give you star on your repository.
